test/test_server_upload.py

```diff
@@ -9,9 +9,15 @@
 import jupyter_cms
 from tornado import web
 
+dashboard_link = 'http://notebook-server:3000/dashboards/test'
+
 class MockResult(object):
     def __init__(self, status_code):
         self.status_code = status_code
+        if ('TEST_DISABLE_UPLOAD_LINK' in os.environ):
```

Comment:
When you asked me about this, I thought you wanted to drive the `DASHBOARD_REDIRECT_URL` env var, not something new. For this use case, you shouldn't be using an env var at all. I think you should just pass whether the upload link is enabled or disabled via the MockResult constructor.